packages/manager/src/features/Linodes/LinodesCreate/LinodeCreateContainer.tsx

```diff
@@ -1018,5 +1023,11 @@ const handleAnalytics = (
     eventLabel = label;
   }
 
-  sendCreateLinodeEvent(eventAction, eventLabel);
+  sendCreateLinodeEvent(
+    eventAction,
+    eventLabel,
+    clonedLinode
+      ? { isLinodePoweredOff: clonedLinode.status === 'offline' }
+      : undefined
```

Comment:
if we only want to send on clone create events, not any create event with an existing linode:
```suggestion
    linode && eventAction === 'clone'
      ? { isLinodePoweredOff: linode.status === 'offline' }
      : undefined
```
Since we have the graph that shows component details vs component clicks and we can see clone events vs any other type of event there, we don't *need* to only collect powered on/off data on clone events, but it does make it more relevant to graph the "total number of linodes powered on/off".